geps/gep-2627/index.md

```diff
@@ -0,0 +1,50 @@
+# GEP-2627: DNS configuration within Gateway API
+
+* Issue: [#2627](https://github.com/kubernetes-sigs/gateway-api/issues/2627)
+* Status: Provisional
+
+## TLDR
+
+For gateway infrastructure to be valuable we need to be able to connect clients to these gateways. A common way to achieve this is to use domain names/hostnames and DNS. Gateways define listeners that can have assigned hostnames or wildcards.  The guidelines for DNS configuration are a critical piece of service networking, but this is currently not expressible as part of Gateway API.   Instead of leaving this as an exercise for the user to figure out, this proposal attempts to provide options to ease Gateway API operations.
+
+## Goals
+* Provide a way for Gateway API resource owners to mark their resources as relevant for external DNS provisioning
+* Ensure that the above method has a way for multiple providers to be present in the cluster and be able to actuate external DNS provisioning requests
+* Ensure that any method is based on structured fields and makes the most of `status` on whatever resources are relevant, whether they are existing Gateway API resources or new resources.
+* Increase portability and supportability between Gateway API implementations and third party controllers offering DNS integration.
+* Clarity on the scope of hostnames under management. (This should _not_ be able to be used to affect the standard in-cluster DNS configuration)
```

Comment:
I was wondering if you might be able to elaborate on this goal a bit further please?

Reply:
So it was a while since I looked at this. I am trying to recall. But I believe this was focused around only using the hosts defined in the gateway and not getting involved with or conflicting with the exsting service name dns IE `data.test.svc.cluster.local`

Reply:
Is it OK to pull it, just so there's no ambiguity?

```suggestion
```

OR do you prefer we keep it (maybe a rewording)?

Reply:
I am ok with removing this 👍